tests/browser/patchright.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+
+from patchright.async_api import async_playwright
+
+
+async def test_patchright_launch_and_close():
+	async with async_playwright() as p:
+		browser = await p.chromium.launch(headless=True)
+		context = await browser.new_context()
+		page = await context.new_page()
+
+		await page.goto('https://example.com')
```

Comment:
dont use live URLs in tests, use pytest-httpserver like in `test_controller.py` or just visit `chrome://version` or `about:blank`